### PR TITLE
Assigment 4; task 6 and 7

### PR DESCRIPTION
**Pull request by angeel2706**

### CODE REVIEW

**Review by dachafra — CHANGES_REQUESTED** [review]
task 6 ok
task7: results are not complete and you have to do the queries ALSO in SPARQL

**Comment by dachafra** [comment]
this PR also contains assignment 2 and 3 @ocorcho

File: Assignment2/angeel2706.ttl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+Class 01: 	includes : Computer 101;
+		includes : Sensor 029.
+
+Sensor 029: 	hasMeasurement : Measurement 8401.
+
+Measurement 8401: hasTemperature: "29";
```
Comment (ocorcho) [inline Assignment2/angeel2706.ttl:7]: Wrong syntax.

**Review by ocorcho — CHANGES_REQUESTED** [review]
Errors in Assignment 2a, and assignment 3 has also errors in queries 4 and 5.

**Comment by ocorcho** [comment]
Closing this since the original repository has been deleted.

**Pull request closed without merging**